package.json

```diff
@@ -23,59 +23,63 @@
-    "build-storybook": "build-storybook",
+    "build-storybook": "storybook build",
     "prerelease": "node check-metadata.js",
     "release": "npm run build && auto shipit"
   },
   "dependencies": {
-    "copy-to-clipboard": "^3.3.1",
-    "core-js": "^3.8.2",
+    "copy-to-clipboard": "^3.3.3",
+    "core-js": "^3.29.0",
     "escape-html": "^1.0.3",
     "fast-deep-equal": "^3.1.3",
     "global": "^4.4.0",
-    "lodash": "^4.17.20",
-    "prop-types": "^15.7.2",
-    "qs": "^6.10.0",
-    "react-colorful": "^5.1.2",
+    "lodash": "^4.17.21",
+    "prop-types": "^15.8.1",
+    "qs": "^6.11.1",
+    "react-colorful": "^5.6.1",
     "react-lifecycles-compat": "^3.0.4",
-    "react-select": "^3.2.0"
+    "react-select": "^5.7.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
-    "@babel/preset-env": "^7.12.1",
-    "@babel/preset-react": "^7.12.5",
-    "@babel/preset-typescript": "^7.13.0",
-    "@emotion/jest": "^11.3.0",
-    "@storybook/react": "^6.4.0",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/user-event": "^13.2.1",
-    "@types/escape-html": "^1.0.1",
-    "@types/lodash": "^4.14.168",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.0",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.0",
+    "@emotion/jest": "^11.10.5",
+    "@storybook/react": "^7.0.0",
+    "@storybook/react-webpack5": "^7.0.0",
+    "@testing-library/dom": "^9.2.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/escape-html": "^1.0.2",
+    "@types/lodash": "^4.14.191",
+    "@types/node": "^18.15.11",
     "@types/react-lifecycles-compat": "^3.0.1",
     "@types/react-select": "^3.1.2",
-    "@types/webpack-env": "^1.16.0",
-    "auto": "^10.3.0",
-    "babel-loader": "^8.1.0",
-    "chalk": "^2.4.2",
-    "concurrently": "^5.3.0",
-    "jest": "^27.0.6",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "rimraf": "^3.0.2",
-    "typescript": "^3.9.7"
+    "@types/webpack-env": "^1.18.0",
+    "auto": "^10.43.0",
+    "chalk": "^5.2.0",
+    "concurrently": "^7.6.0",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "rimraf": "^4.4.0",
+    "storybook": "^7.0.0",
+    "typescript": "^4.9.5"
   },
   "peerDependencies": {
-    "@storybook/addons": "^6.4.0",
-    "@storybook/api": "^6.4.0",
-    "@storybook/components": "^6.4.0",
-    "@storybook/core-events": "^6.4.0",
-    "@storybook/theming": "^6.4.0",
-    "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "@storybook/addons": "7.0.0",
+    "@storybook/api": "7.0.0",
+    "@storybook/components": "7.0.0",
+    "@storybook/core-events": "7.0.0",
+    "@storybook/theming": "7.0.0",
```

Comment:
Hard pinning these to "7.0.0" breaks peer dependencies for any project that upgrades to any patch releases like the current 7.0.7. Shouldn't these be caret pinned instead to "^7.0.0"?

Reply:
I opened [this PR](https://github.com/storybookjs/addon-knobs/pull/51). Thanks for the heads up! I missed this